runelite-client/src/main/java/net/runelite/client/plugins/keyremapping/KeyRemappingConfig.java

```diff
@@ -276,14 +277,34 @@ default ModifierlessKeybind esc()
 		return new ModifierlessKeybind(KeyEvent.VK_ESCAPE, 0);
 	}
 
+	@ConfigSection(
+		name = "Miscellaneous Remapping",
```

Comment:
I don't think a new section for this is required. Having no section usually implies miscellaneous.